# artemis_results
#Results of the artemis variant

This analysis explores variants of the protein artemis, focusing on its different isoforms and their potential functional implications. The key question addressed is whether the shorter isoforms retain residual function - even though they lack part of the active site. This inquiry is supported by a functional assay [https://doi.org/10.1016/j.jaci.2015.03.005], which likely provides insights into the functional capabilities of these isoforms.

Furthermore, the analysis includes AlphaMissense predictions of all mutations of the canonical transcript. Notably, Clinvar pathogenic and likely pathogenic missense mutations are located mostly along exons 1 through 4.

# The 692 residue (Isoform A) (left) and 434 residue (right) isoforms. Both containing exons 1 through 4. 

<p float="left">
  <img src="https://github.com/izzetbiophysicist/artemis_results/blob/main/isoform9.png" width="400" />
  <img src="https://github.com/izzetbiophysicist/artemis_results/blob/main/isoform10.png" width="400" /> 
</p>

# The 577 residue (Isoform B) (left) and 572 residue (Isoform C) (right) isoforms. Both of which lack exons 1 through 4. 

<p float="left">
  <img src="https://github.com/izzetbiophysicist/artemis_results/blob/main/isoform5.png" width="400" />
  <img src="https://github.com/izzetbiophysicist/artemis_results/blob/main/isoform8.png" width="400" /> 
</p>


# Do shorter isoforms retain residual function?

https://doi.org/10.1016/j.jaci.2015.03.005 - isoforms functional assay 

![Alt Text](https://github.com/izzetbiophysicist/artemis_results/blob/main/function.png)

# AlphaMissense predictions of all mutations of the canonical transcript
Clinvar pathogenic and likely pathogenic missense mutations are marked in green
![Alt Text](https://github.com/izzetbiophysicist/artemis_results/blob/main/heat_map.png)
